lib/jarvis/git.rb

```diff
@@ -11,6 +11,13 @@ def self.clone_repo(git_url, workdir)
     ::Git.clone(git_url, name, :path => workdir)
   end
 
+  # return the last sha1 of a repo
+  def self.sha1(workdir)
```

Comment:
I think the Git library can already do this. If you want the commit hash of HEAD, `git.revparse("HEAD")`